src/core/server/metrics/ops_metrics_collector.ts

```diff
@@ -28,14 +28,16 @@ export class OpsMetricsCollector implements MetricsCollector<OpsMetrics> {
   }
 
   public async collect(): Promise<OpsMetrics> {
-    const [process, os, server] = await Promise.all([
+    const [processes, os, server] = await Promise.all([
       this.processCollector.collect(),
       this.osCollector.collect(),
       this.serverCollector.collect(),
     ]);
+
     return {
       collected_at: new Date(),
-      process,
+      process: processes[0],
+      processes,
```

Comment:
NIT: add a comment explaining that atm `processes` is just an Array(1) only returning the current process's data, which is why we can just `process: processes[0]`